certif/app/components/add-student-list.js

```diff
@@ -75,11 +75,11 @@ export default class AddStudentList extends Component {
     try {
       await this.args.session.save({ adapterOptions: { studentListToAdd, sessionId } });
       this.args.returnToSessionCandidates(sessionId);
-      this.notifications.success('Le(s) candidat(s) ont été inscrit(s) avec succès.');
+      this.pixToast.sendSuccessNotification({ message: 'Le(s) candidat(s) ont été inscrit(s) avec succès.' });
```

Comment:
Question: On n'utilise pas la trad ici ? `this.intl.t('pages.sessions.import.candidates-list.import-success')`